lib/modules/showImages.js

```diff
@@ -833,19 +833,11 @@ const retrieveExpandoOptions = _.memoize(async (thing, { siteModule, detectResul
 
 	const isMuted = ({ muted, type }) => muted || ['IMAGE', 'TEXT'].includes(type);
 
-	if (mediaOptions.type === 'GALLERY') {
-		mediaOptions.muted = mediaOptions.src.every(isMuted);
-
-		if (mediaOptions.src.length === 1) {
-			mediaOptions.type = 'SINGLE_PIECE_GALLERY';
-			mediaOptions.expandoClass = mediaOptions.src[0].type.toLowerCase();
-		}
-	}
-
 	mediaOptions = {
 		attribution,
+		get buttonInfo() { return getMediaButtonInfo(mediaOptions); },
```

Comment:
If this doesn't need to be reevaluated each time (and because AFAICT it's always or almost always used), then assign it normally (no getter) after this assignment, i.e.  after line 842.